utils/data_preprocessing.py

```diff
@@ -194,9 +205,10 @@ def load_data(dir: str, unwanted_keywords: Set[str]):
     vocabulary = []
     overall = []
     for file in files:
+
         meta, content = get_clean_content(file)
         if content is not None:
-            vocabulary.extend(content)
+            vocabulary.append(content)
```

Comment:
You sure? `content: List[str]` so `vocab` now becomes `List[List[str]]` where `len(vocab) == 1` if I am correct